test/end2end/app_test.dart

```diff
@@ -158,7 +160,8 @@ Future<void> bottomNavigation(WidgetTester tester) async {
   // Map
   await tester.tap(find.text("Map"));
   await tester.pumpAndSettle();
-  expect(find.text("Proxima"), findsOneWidget);
+  expect(find.byType(MapScreen), findsOneWidget);
+  expect(find.byType(PostMap), findsOneWidget);
```

Comment:
I think that only checking that `MapScreen` is present is enough here because this test only checks at a high level that elements are displayed.
This is only my interpretation though, so what do you think ?

```suggestion
```

Reply:
I understand your point of view, but I think that there's never enough testing. In my opinion, it is also important to check that map itself was loaded correctly. Even if we're at high level, I think that assuring that the map screen indeed displays a map is crucial.

Reply:
Ok I understand, then we can let it like that.